.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@
 ---
 default_stages: [pre-commit, pre-push]
 default_language_version:
-  python: python3
+  python: python3.12
```

Comment:
Is that related to this PR ?

Reply:
No. Had issues with pre-commit. Will remove it when ready